Rakefile

```diff
@@ -1,7 +1,42 @@
 require 'rake/testtask'
 
 Rake::TestTask.new do |t|
-  t.pattern = "test/**/*_test.rb"
+	t.pattern = "test/**/*_test.rb"
+end
+
+task :environment do
+	if !ENV["RACK_ENV"] || ENV["RACK_ENV"] == "development"
+		require "dotenv"
+		Dotenv.load
```

Comment:
Maybe just try to load dotenv, and only use if installed? Instead of assuming the rack_env will match how the bundle was installed?